packages/replayio/src/utils/getReplayPath.ts

```diff
@@ -0,0 +1,16 @@
+import assert from "assert";
+import { join, resolve } from "path";
+
+export function getReplayPath(...path: string[]) {
+  let basePath;
+  if (process.env.RECORD_REPLAY_DIRECTORY) {
+    basePath = process.env.RECORD_REPLAY_DIRECTORY;
```

Comment:
I was told before that everything with `RECORD_REPLAY_*` is considered to be legacy-ish. I guess that this one isn't often used by the user but it would be cool to have a consistent naming scheme for all of those options-like env vars on our side